lanebreach-api/app/models/sf311_case_metadatum.rb

```diff
@@ -0,0 +1,29 @@
+# == Schema Information
+#
+# Table name: sf311_case_metadata
+#
+#  id             :bigint(8)        not null, primary key
+#  bike_lane_type :string
+#  created_at     :datetime         not null
+#  updated_at     :datetime         not null
+#  sf311_case_id  :bigint(8)
+#
+# Indexes
+#
+#  index_sf311_case_metadata_on_sf311_case_id  (sf311_case_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (sf311_case_id => sf311_cases.id)
+#
+
+class Sf311CaseMetadatum < ApplicationRecord
+  belongs_to :sf311_case
+
+  def self.create_metadata(sf311_case)
+    bikeway_network = BikewayNetwork.nearest(sf311_case.lat, sf311_case.long)
+    if bikeway_network.present?
+      Sf311CaseMetadatum.create!(:bike_lane_type => bikeway_network.symbology, :sf311_case => sf311_case)
```

Comment:
This is nitpicky, but I think we should standardize on Ruby 2 hash syntax (i.e., `bike_lane_type: bikeway_network.symbology`)  to keep code style consistent.

Reply:
Totally, good call. I imagine eventually we could add a linter too that can make these checks.